earth_enterprise/BUILD_RHEL_Centos.md

```diff
@@ -0,0 +1,133 @@
+# GEE Build Setup for CentOS, RHEL 6 and RHEL 7     
+
+
+## Install git  
+    
+Install the system default version:
+    
+```
+sudo yum install git
+```
+    
+Or install the latest version from the IUS repo (recommended) [ [More Info] ](https://ius.io/GettingStarted/):
+```
+sudo yum install -y wget
+cd /tmp
+wget https://rhel7.iuscommunity.org/ius-release.rpm
+sudo yum install ius-release.rpm
+sudo yum install git2u-all
+```
+        
+        
+## Install git lfs
+   
+Execute the following commands
+    
+```
+curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.rpm.sh | sudo bash
+sudo yum install git-lfs
+```
+
+
+
+## Enable additional repos
+Different yum repositoies need to be enabled for different distributions to install several required development tools.  
+
+### CentOS 7 
+```
+sudo yum install epel-release
+```
+
+### RHEL 7  
+
+_note:_ If the first command doesn't work, try the alternate methods listed [here](https://www.cyberciti.biz/faq/installing-rhel-epel-repo-on-centos-redhat-7-x/)
+```
+sudo subscription-manager repos --enable=rhel-7-server-optional-rpms
+sudo subscription-manager repos --enable=rhel-7-server-optional-source-rpms
+```    
+
+### RHEL 6
+```
+sudo subscription-manager repos --enable=rhel-x86_64-workstation-dts-2
+sudo subscription-manager repos --enable=rhel-6-server-optional-rpms
+sudo subscription-manager repos --enable=rhel-6-server-optional-source-rpms
+```
+       
+## GCC 4.8
+
+### CentOS 7 and RHEL 7
+
+To obtain GCC 4.8 on RHEL6, you must enable the "Development Tools" repository in the RHSM: 
+```
+sudo yum --setopt=group_package_types=mandatory,default,optional groupinstall "Development Tools"
+sudo yum install scons perl-Perl4-CoreLibs xorg-x11-server-devel python-devel perl-Alien-Packages gtest-devel openssl-devel libxml2-devel libXinerama-devel libXft-devel libXrandr-devel libXcursor-devel gdbm-devel libmng-devel libcap-devel libpng12-devel libXmu-devel freeglut-devel zlib-devel libX11-devel bison-devel openjpeg-devel openjpeg2-devel geos-devel proj-devel ogdi-devel giflib-devel xerces-c xerces-c-devel
```

Comment:
This is a repeat of the command listed at the bottom of the page

Reply:
Dang - thought I'd fixed that already - will fix!